geoalchemy2/elements.py

```diff
@@ -136,6 +137,27 @@ class WKBElement(_SpatialElement):
     geom_from = 'ST_GeomFromWKB'
     geom_from_extended_version = 'ST_GeomFromEWKB'
 
+    WKB_XDR = 0  # Most Significant Byte First
+    WKB_NDR = 1  # Least Significant Byte First
+
+    def __init__(self, data, srid=-1, extended=False):
+        if extended and srid == -1:
+            # unpack srid from WKB struct
+
+            try:
+                # support hex input
+                data = binascii.unhexlify(data)
```

Comment:
maybe use `self._data_from_desc` ?

Reply:
finally I used slicing to not damage perf with big geometries